storage/object/api-cli/bucket-policy.mdx

```diff
@@ -605,4 +605,4 @@ Bucket policies use a JSON-based access policy language and are composed of stri
 - `DateLessThan`
 - `DateLessThanEquals`
 
-
+Refer to our [dedicated documentation](/storage/object/api-cli/manage-bucket-permissions-IP/) for more information on managing bucket permissions for IP addresses or ranges of IP.
```

Comment:
Posssible to rename this to `/storage/object/api-cli/manage-bucket-permissions-ip/` ?
would be better not to have uppercase characters in the slug